### PR TITLE
bash-completion: version bumped to 2.4.

### PR DESCRIPTION
**Pull request by Florin65**

### CODE REVIEW

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by lunar-linux-b01** [comment]
SUCCESS!

File: utils/bash-completion/POST_BUILD [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# relin util-linux because some completions are overlaping with bash-completion
+lin -c util-linux
```
Comment (Ratler) [inline utils/bash-completion/POST_BUILD]: @Florin65 this is not a good idea, we should resolve the conflict. Won't merge this.

File: utils/bash-completion/DETAILS [hunk 2]
```diff
@@ -1,21 +1,13 @@
           MODULE=bash-completion
-         VERSION=2.1
-          SOURCE=${MODULE}-${VERSION}.tar.bz2
-         SOURCE2=${MODULE}-1.99-lunar_fixes.patch
```
Comment (Ratler) [inline utils/bash-completion/DETAILS]: This patch is somewhat valid still.

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by Florin65** [comment]
OK, i have fixed the issues.

**Comment by Ratler** [comment]
Thank you. Looks fine.

**Pull request merged**